doc(release) Release notes collect 23.04.15 / gorgone 23.04.16 (#3609)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx

## Centreon Collect

### 23.04.15

Release date: `September 30, 2024`

<details>
  <summary>Bug fixes</summary>

> - [Broker] If a service contains several times the same metric name, some of them were overwritten. Now during the parsing, a warning log explains the issue and only the first occurence of the metric is kept.

Reviewer: ```suggestion
- [Broker] If a service contained several times the same metric name, some of them were overwritten. Now during the parsing, a warning log explains the issue and only the first occurrence of the metric is kept.
```

> - [Broker] If a service contained several times the same metric name, some of them were overwritten. Now during the parsing, a warning log explains the issue and only the first occurrence of the metric is kept.

Reviewer: ```suggestion
- [Broker] If a plugin sends the same metric several times, an error log from broker explains the issue and only the first occurrence of the metric is kept.
```

> - [Engine] Fixed issues on external commands output.

Reviewer: ```suggestion
- [Engine] Fixed issues with the output of external commands.
```
